modules/programs/getmail-accounts.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, ... }:
+
+with lib;
+
+{
+  options.getmail = {
+    enable = mkEnableOption "the getmail mail retriever for this account";
+
+    destinationCommand = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      example = "\${pkgs.maildrop}/bin/maildrop";
+      description = ''
+        Specify a command delivering the incoming mail to your maildir.
+      '';
+    };
+
+    mailboxes = mkOption {
+      type = types.listOf types.str;
+      default = [];
+      example = ["INBOX" "INBOX.spam"];
+      description = "A list of mailboxes";
```

Comment:
Descriptions should end in a `.` (the `mkEnableOption` argument is an exception since the `.` is added by the function).

Reply:
Done.